templates/cloudformation/apps/aem/full-set/monitoring.yaml

```diff
@@ -15,37 +15,429 @@ Resources:
+                ]
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 39,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  [ "collectd", "GenericJMX.delay.seconds_since_last_success", "FixedDimension", "${stackPrefix}-author-standby", "Host", "${authorStandbyInstance}", "PluginInstance", "standby-status", { "period": 60, "stat": "Maximum" } ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "start": "-PT3H",
+                "end": "P0D",
+                "title": "GenericJMX.delay.seconds_since_last_success"
+              }
+            }
+          ]
         }'
         - {
           authorDispatcherAutoScalingGroupName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-AuthorDispatcherAutoScalingGroup'}]],
           publishAutoScalingGroupName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-PublishAutoScalingGroup'}]],
           publishDispatcherAutoScalingGroupName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-PublishDispatcherAutoScalingGroup'}]],
           orchestratorAutoScalingGroupName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-OrchestratorAutoScalingGroup'}]],
+#          choasMonkeyAutoScalingGroupName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-ChaosMonkeyAutoScalingGroup'}]],
+          choasMonkeyAutoScalingGroupName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-OrchestratorAutoScalingGroup'}]],
+          authorPrimaryInstance: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-AuthorPrimaryInstance'}]],
+          authorStandbyInstance: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-AuthorStandbyInstance'}]],
+
+          authorELB: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-AuthorLoadBalancer'}]],
+          authorDispatcherELB: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-AuthorDispatcherLoadBalancer'}]],
+          publishDispatcherELB: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-PublishDispatcherLoadBalancer'}]],
+
+          aemAsgEventQueueName: !Join ['', [ "Fn::ImportValue": {"Fn::Sub":'${MainStackPrefixParameter}-AEMASGEventQueueName'}]],
+
+          stackPrefix: !Ref MainStackPrefixParameter,
           awsRegion: !Ref AwsRegionParameter
         }
       DashboardName: !Sub
         - '${MainStackPrefixParameter}-monitoring-dashboard'
         - {
             MainStackPrefixParameter: !Ref MainStackPrefixParameter
           }
+
+
+Outputs:
+  applicationCloudwatchDashboardURL:
```

Comment:
Other stacks' outputs keys seem to be using title case. While `applicationCloudWatchDashboardURL` is camel case. Can this please be made consistent?

Reply:
replaced with `ApplicationCloudWatchDashboardURL`.